String_Exercises.js

```diff
@@ -0,0 +1,185 @@
+//1. `console.log` the first character of a string.
```

Comment:
Overall good demonstration of string methods and properties. I like the fact that you tried different methods and statements to solve problems. Good job and keep up the good work!